src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -226,6 +251,34 @@ export default class TrafficRateController {
         return $element;
     }
 
+    /**
+     * Build form element
+     *
+     * @for TrafficRateController
+     * @method _buildWindSlider
+     * @param key {string}
+     * @param value {number} passed to the change handler
+     * @param onChangeMethod {function}
+     * @param min {int} min value of slider
+     * @param max {int} max value of slider
+     * @param step {int} step value of slider
+     * @return {jquery|HTML Element}
+     */
+    _buildWindSlider(key, label, value, onChangeMethod, min, max, step) {
+        const template = `
+            <div class="form-element">
+                <div class="form-label">${label}</div>
+                <input class="form-slider" type="range" name="${key}" value="${value}" min="${min}" max="${max}" step="${step}" />
+                <span class="form-value">${value}</span>
+            </div>`;
+        const $element = $(template);
+        const onChangeHandler = onChangeMethod.bind(this);
```

Comment:
please bind this to the class as a handler within the `_setupHandlers` method:
```js
_setupHandlers() {
  this._onChangeWindDirectionHandler = this.onChangeWindDirection.bind(this);

  return this;
}
```

Reply:
@n8rzz I followed the same pattern used here for the rate sliders, since the direction and speed handlers are different events. this prevents duplicate code by allowing me to pass the handler into the creation of the element as a param. I was under the impression based on patterns used elsewhere that _setupHandlers() was meant for eventBus events, and not for gui element events. If I make the proposed changes, I would end up having to add another buildXSlider method for the speed. Thoughts?

Reply:
`setupHandlers` is there to bind any handlers to `this`.  those handlers might be used as EventBus callbacks, dom event handlers, etc.  Its a single place to do all the binding.

I see what you mean, though, we are doing it like this elsewhere.  gross.  I'll leave it up to you; if you want to make the change, awesome.  If not, we can leave it.